processing/WordVectors.py

```diff
@@ -0,0 +1,126 @@
+import json
+import random
+import re
+import sys
+
+import matplotlib.pyplot as plt
+import numpy
+import spacy
+from gensim.parsing.preprocessing import preprocess_string, remove_stopwords
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+
+
+def get_words(text):
+    """
+    Source: <@FIXME:/somewhere/>
+    """
+    text = text.lower()
+    wordlist = text.split()
+    clean_list = []
+    for word in wordlist:
+        # only get words (no digits)
+        if not word.isdigit() and not re.match(r"[^\w]", word):
+            clean_list.append(word)
+    return " ".join(clean_list)
+
+
+class WordVectors:
+    """
+    @FIXME: class to represent word vectors. more stuff here
+    """
+
+    dataset = []  # list of dictionaries with "summary" and "_id" fields
+    # @FIXME: or perhaps we ask for a more general input format?
```

Comment:
make this more general; perhaps use `{"summary" : <summary>, "theme" : <theme> 🔢}`